docs/itemset_dbc.md

```diff
@@ -0,0 +1,795 @@
+# itemset_dbc
+
+[<-Back-to:World](database-world)
+
+**The \`itemset_dbc\` table**
+
+**Table Structure**
+
+| Field                                   | Type    | Attributes | Key | Null | Default | Extra | Comment |
+| --------------------------------------- | ------- | ---------- | --- | ---- | ------- | ----- | ------- |
+| [ID](#id)                               | INT     | SIGNED     | PRI | NO   | 0       |       |         |
+| [Name_Lang_enUS](#name_lang_enus)       | VARCHAR | SIGNED     |     | YES  | NULL    |       |         |
```

Comment:
FYI these links won't work. in a relative link to a header you cannot use special characters.

so a link to this header would be `#namelangenus`

```
## Name_Lang_enUS

(#name_lang_enus) -> Dont work
(#namelangenus) -> Work